src/main/config/configuration.properties

```diff
@@ -20,3 +20,7 @@ registry = https://plugins.dita-ot.org/
 
 # PDF2 defaults
 org.dita.pdf2.i18n.enabled = true
+
+# Compatibility modes
+# Treat keyref elements that contain only whitespace as empty.
```

Comment:
@infotexture This explanation needs some expansion. Are you able to improve this description:

Value `false` is correct behaviour per DITA spec. Value `true` is not valid behaviour per DITA spec, but it currently used for backwards compatibility to not break existing content.

Reply:
Or should we just say here that we use `true` for backwards compatibility and will change in future. Kinda like `XXX: ` prefixed comment. Then in docs describe this property and how it works?